Portfolio milestone 04

shahadat3669 opened the pull request:
In this milestone in my portfolio website, I finished:

- [x] Used media queries to make sure that my website displays correctly on desktop screens.
- [x] To lay out the elements on the page I used Grid and Flexbox
- [x] Replicated the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Yazino12 commented on index.html:
```diff
@@ -20,6 +17,11 @@
           <nav class="header-nav">
             <a href="/" class="header-logo">Welcome</a>
+            <ul class="nav__list">
+              <li class="nav__item">Portoflio</li>
+              <li class="nav__item">About</li>
+              <li class="nav__item">Contact</li>
```
- [x] Please consider using an `<a>` tag for your navigation buttons 🦋, that way users can easily navigate through your page with a click of a button 📎. **_You will end up with something like this_** 👇🏽

![image](https://user-images.githubusercontent.com/43172164/200037646-92c66d03-e047-4a3f-9b91-097c3a0f9d75.png)
**_Please don't forget to style the anchor tags after adding so you stick to the design as much as possible_**

Yazino12 commented on index.html:
```diff
@@ -323,113 +246,59 @@ <h3 class="card__title">Skills</h3>
+            <input type="email" placeholder="Email address" id="contact-form__email" name="email" required />
+            <textarea name="message" id="contact-form__message" cols="30" rows="5" placeholder="Enter text here" maxlength="500" required></textarea>
+            <div class="contact-form__btn-wrapper">
+              <button class="btn btn--primary">Get in touch</button>
```
- [x] The form button is **center-aligned for mobile screens** and **left-aligned for desktop screens** 📝, but in your design, it's **left-aligned** for both `mobile` and `desktop` screens. Kindly fix this 👇🏽:

### Your design:
![image](https://user-images.githubusercontent.com/43172164/200039987-ad68bc12-f98b-48c2-9d3c-8e6e3a8d8905.png)

### Figma design:
![image](https://user-images.githubusercontent.com/43172164/200040136-88ce7781-8b8d-47bb-837e-cee95f1e2f96.png)

Yazino12 reviewed (changes requested):
## Hi @shahadat3669 🖐🏾,

You've come a long way and you are almost there 👏🏽, but there are still some changes you need to work on to move to the next project 🔨.

### To highlight:
- Linters are passing ✔️
- Correct use of GitHub flow ✔️ 
- Descriptive Readme ✔
- Good PR summary ✔

# STATUS:  REQUIRED CHANGES ♻♻♻ 
![THIS CLOSE](https://media.giphy.com/media/z3piokwf0WPH81MOhu/giphy.gif)
_Check the comments under the review 👇🏽👇🏽👇🏽._

- [x] From now on, try to write better commit messages. The sentences should be capitalized and in the imperative form. Example: "Fix background" and not "fixing background" 👍🏽.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Yazino12 when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

shahadat3669 commented:
- [x] Fixed issues with the nav item 

![image](https://user-images.githubusercontent.com/55840999/200043461-92875f4b-0c8a-43fb-a489-952c6ebe458e.png)

- [x] Fixed contact  alignment
![image](https://user-images.githubusercontent.com/55840999/200043691-824a365d-995e-4f8b-8446-2309397ce1eb.png)

aimalamiri reviewed (approved):
Hi @shahadat3669, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![ApprovedDavieGIF](https://user-images.githubusercontent.com/34779928/191333120-c0c2ce83-28e1-4dbc-aac1-09e467f7a8bd.gif)


## Highlights

- The desktop version is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
